internal/org/repository/eventsourcing/org.go

```diff
@@ -0,0 +1,195 @@
+package eventsourcing
+
+import (
+	"context"
+
+	"github.com/caos/zitadel/internal/errors"
+	es_models "github.com/caos/zitadel/internal/eventstore/models"
+	org_model "github.com/caos/zitadel/internal/org/model"
+	"github.com/sony/sonyflake"
+)
+
+var idGenerator = sonyflake.NewSonyflake(sonyflake.Settings{})
```

Comment:
i would move this to OrgEventstore

Reply:
at the moment i will leave it as it is. My next task is to generalise the id-generator